src/EditorFeatures/CSharpTest/Diagnostics/UpgradeProject/UpgradeProjectTests.cs

```diff
@@ -212,20 +191,19 @@ public class Program
 }
 ",
                 LanguageVersion.CSharp7_1,
-                new CSharpParseOptions(LanguageVersion.CSharp7),
-                index: 0);
+                new CSharpParseOptions(LanguageVersion.CSharp7));
         }
 
-#region C# 7.3
+        #region C# 7.3
         [Fact]
-        public async Task UpgradeProjectFromCSharp7_3ToLatest()
+        public async Task UpgradeProjectFromCSharp7_2ToLatest()
```

Comment:
Nit: Test name still seems wrong after latest change.

Reply:
➡️ Test name is correct